src/__experimental__/components/textbox/textbox.component.js

```diff
@@ -51,6 +54,11 @@ const Textbox = ({
           { leftChildren }
           { prefix ? <StyledPrefix data-element='textbox-prefix'>{ prefix }</StyledPrefix> : null }
           <Input
+            required={ required }
+            aria-required={ props['aria-required'] }
+            onInvalid={ (e) => {
```

Comment:
What is `onInvalid`/`setCustomValidity` used for?

Reply:
to avoid to see the default HTML 5 validator tooltip if we use the required tag (I've asked a previous question on it on the channel).

Reply:
ah I see now, in that case I don't think we should need `setCustomValidity` because if you are using HTML5 validation you will want the message.

Reply:
Done